#
# Description: Security fix to prevent the possibility of 
# attack with the help of symlinks in some Debian packages.
# Bug-Debian: http://bugs.debian.org/496398
# Author: Dario Minnucci <midget@debian.org>
#
diff -urNad mon-1.2.0~/alert.d/test.alert mon-1.2.0/alert.d/test.alert
--- mon-1.2.0~/alert.d/test.alert	2009-09-05 09:41:39.504231782 +0200
+++ mon-1.2.0/alert.d/test.alert	2009-09-05 09:43:03.948246674 +0200
@@ -1,4 +1,4 @@
 #!/bin/sh
 #
 # $Id: test.alert,v 1.1.1.1 2004/06/09 05:18:07 trockij Exp $
-echo "`date` $*" >> /tmp/test.alert.log
+echo "`date` $*" >> /var/log/mon/test.alert.log
